source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -398,6 +398,29 @@ def output(self, key, obj):
         return response
 
 
+class MultiPolyGeoJson(fields.Raw):
+    def __init__(self, **kwargs):
+        super(MultiPolyGeoJson, self).__init__(**kwargs)
+
+    def format(self, value):
+
+        polys = []
+
+        for poly in value.polygons:
+            outer = []
+            outer.append([[c.lon, c.lat] for c in poly.outer.coordinates])
```

Comment:
you can do this in one line:

outer = [[c.lon, c.lat] for c in poly.outer.coordinates]]